third_party/README.md

```diff
@@ -112,3 +112,16 @@ Apart from the above steps, please perform the following two steps to generate t
 
 1. Bump the version in the `tools/distrib/python/xds_protos/setup.py`;
 2. Run `tools/distrib/python/xds_protos/build_validate_upload.sh` to upload the built wheel.
+
+### Updating third_party/upb
+
+Since upb is vendored in the gRPC repo, you cannot use submodule to update it. Please follow the steps below.
+
+1. Update third_party/upb directory by running
+   `git subtree pull --squash --prefix=third_party/upb https://github.com/protocolbuffers/upb.git master`
+2. Update the dependency in `grpc_deps.bzl` to the same commit
+3. Populate the bazel download mirror by running `bazel/update_mirror.sh`
+4. Update `src/upb/gen_build_yaml.py` for newly added or removed upb files
+5. Run `tools/buildgen/generate_projects.sh` to regenerate the generated files
+6. Run `tools/codegen/core/gen_upb_api.sh` to regenerate upb files.
+   If you see breaking changes here, you may want to import upb into Google3 along with gRPC.
```

Comment:
We could mention the new presumit process here.

Reply:
I didn't want to expose too much detail here as to internal import process. Usually it requires a manual CL by merging two import CLs so I cannot imagine this can be done automatically by the new presubmit process.